packages/devtools_app/lib/src/primitives/enum_utils.dart

```diff
@@ -32,3 +32,24 @@ class EnumUtils<T> {
 
   String? name(T enumEntry) => _reverseLookupTable[enumEntry];
 }
+
+mixin EnumIndexOrdering<T extends Enum> on Enum implements Comparable<T> {
+  @override
+  int compareTo(T other) => index - other.index;
```

Comment:
is index an int? `=> index.compareTo(other.index)` might be more clear

Reply:
Done!